lfs/client.go

```diff
@@ -125,6 +126,58 @@ func (b *byteCloser) Close() error {
 	return nil
 }
 
+type upload struct {
+	OIDPath  string
+	Filename string
+	CB       CopyCallback
+}
+
+type UploadQueue struct {
+	uc     chan upload
+	ec     chan *WrappedError
```

Comment:
Yeah, I'm working on renaming some things, my rough pass names tend to be ... rough :smile: